lockfile/src/javascript.rs

```diff
@@ -273,7 +273,7 @@ mod tests {
             name: "@yarnpkg/lockfile".to_string(),
             version: "1.1.0".to_string(),
             package_type: PackageType::Npm,
-        }]);
```

Comment:
Did you remove this semicolon? I don't think `rustfmt` should do this.

Reply:
I was fixing them manually but both `cargo +nightly fmt --all -- --check` and `cargo build` seem to be fine with it, I'll add it back just in case.